File: magellan-sample/src/main/java/com/wealthfront/magellan/sample/DetailScreen.java
package com.wealthfront.magellan.sample;

import android.content.Context;

import com.wealthfront.magellan.Screen;

class DetailScreen extends Screen<DetailView> {

    @Override
    protected DetailView createView(Context context) {
        return new DetailView(context);
    }

    @Override
    public String getTitle(Context context) {
        return "Detail Screen";
    }

}

File: magellan-sample/src/main/java/com/wealthfront/magellan/sample/DetailScreen.kt
package com.wealthfront.magellan.sample

import android.content.Context

import com.wealthfront.magellan.Screen

internal class DetailScreen : Screen<DetailView>() {

    override fun createView(context: Context): DetailView {
        return DetailView(context)
    }

    override fun getTitle(context: Context): String {
        return "Detail Screen"
    }

}
